// @smartsupp/websocket-client-types@6.6.1/lib/index.js
"use strict";
Object.defineProperty(exports, "__esModule", {
    value: true
});
exports.MessageDeliveryFailReason = exports.MessageDeliveryStatus = exports.AgentDevicePlatform = exports.AgentDeviceType = exports.VisitorStatus = exports.ChatReadInfoType = exports.ChatStatus = exports.AgentStatus = exports.AccountStatus = void 0;
exports.AccountStatus = {
    Online: 'online',
    Offline: 'offline',
    OfflineWithAI: 'offlineWithAi',
};
exports.AgentStatus = {
    Online: 'online',
    Offline: 'offline',
};
exports.ChatStatus = {
    Pending: 'pending',
    Open: 'open',
    Closed: 'closed',
    Served: 'served',
};
exports.ChatReadInfoType = {
    Agent: 'agent',
    Contact: 'contact',
};
exports.VisitorStatus = {
    Active: 'active',
    Clicked: 'clicked',
    Idle: 'idle',
    Served: 'served',
    Unserved: 'unserved',
    Triggered: 'triggered',
};
exports.AgentDeviceType = {
    Browser: 'browser',
    Mobile: 'mobile',
    Xmpp: 'xmpp',
    Other: 'other',
};
exports.AgentDevicePlatform = {
    IOS: 'ios',
    ANDROID: 'android',
};
exports.MessageDeliveryStatus = {
    Ok: 'ok',
    PermanentFail: 'permanent_fail',
    TemporaryFail: 'temporary_fail',
    Complained: 'complained',
    Seen: 'seen',
};
exports.MessageDeliveryFailReason = {
    Facebook24hWindow: 'facebook.outside_allowed_window',
    WhatsappUnsupportedFile: 'whatsapp.unsupported_file_type',
    WhatsappUnknown: 'whatsapp.unknown',
};
